apps/aeutils/priv/extensions/rename_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Wrapper to run rename_db
+
+PATH=$BINDIR:$PATH
+export ERL_LIBS=$ROOTDIR/lib
+APPS_VSN=${REL_VSN:?}
+
+"$ERTS_DIR/bin/escript" "$ROOTDIR/lib/aeutils-${APPS_VSN:?}/priv/rename_db" $*
```

Comment:
```
$ ./bin/aeternity rename_db '/Users/luca/dev/aeternity/tmp/test-db-renaming/2/I love spaces'
Starting node name renaming in the db from "love" to "spaces"
Schema file not found: "/Users/luca/dev/aeternity/tmp/test-db-renaming/2/I/mnesia/schema.DAT"
Script changing node in the db failed: {error,file_not_found}
```
Exit status: 1.

```suggestion
"$ERTS_DIR/bin/escript" "$ROOTDIR/lib/aeutils-${APPS_VSN:?}/priv/rename_db" "$@"
```

```
$ ./bin/aeternity rename_db '/Users/luca/dev/aeternity/tmp/test-db-renaming/2/I love spaces'
Starting node name renaming in the db from "epoch@localhost" to "aeternity@localhost"
Created schema.DAT backup file at: "/Users/luca/dev/aeternity/tmp/test-db-renaming/2/I love spaces/mnesia/schema.DAT.backup". Please restore from it if renaming process is interrupted or fails.

=INFO REPORT==== 13-Mar-2019::13:47:13 ===
disk_log: repairing "/Users/luca/dev/aeternity/tmp/test-db-renaming/2/I love spaces/mnesia/LATEST.LOG" ...
Hurray! Node name renaming in the db was successful! You're good to start your node now!
```
Exit status: 0.

Reply:
I am not sure re Win separate script.